src/lib/full-path.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Get a PostCSS source full path
+ * @module lib/full-path
+ */
+
+const {join, dirname} = require('path');
+
+module.exports = (fp, rp) =>
```

Comment:
I always think more compact = more readable. Anyway I will not change your PR if you don't want to. :D